bin/yarn

```diff
@@ -3,7 +3,7 @@ APP_ROOT = File.expand_path('..', __dir__)
 Dir.chdir(APP_ROOT) do
   yarn = ENV["PATH"].split(File::PATH_SEPARATOR).
     select { |dir| File.expand_path(dir) != __dir__ }.
-    product(["yarn", "yarn.exe"]).
+    product(["yarn", "yarn.cmd", "yarn.ps1"]).
```

Comment:
what was this for? was it because of any issues you had?

Reply:
It got modified when i installed a new gem, probably faker or image processing?